src/components/TodoList/TodoList.tsx

```diff
@@ -1,100 +1,57 @@
+/* eslint-disable max-len */
 import React from 'react';
+import { Todo } from '../../types/Todo';
 
-export const TodoList: React.FC = () => (
-  <table className="table is-narrow is-fullwidth">
-    <thead>
-      <tr>
-        <th>#</th>
-        <th>
-          <span className="icon">
-            <i className="fas fa-check" />
-          </span>
-        </th>
-        <th>Title</th>
-        <th> </th>
-      </tr>
-    </thead>
+interface Props {
```

Comment:
Same here:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/87986955/9826bfed-af74-436e-9f3e-002b53d941b5)